runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1200,6 +1201,12 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          client->write(response, fe->inSnapshotMode());
          }
          break;
+      case MessageType::VM_isPortableRestoreMode:
+         {
+         client->getRecvData<JITServer::Void>();
+         client->write(response, fe->isPortableRestoreModeEnabled());
+         }
+         break;
```

Comment:
I don't think you need to have this message. We're already caching the result of `isJVMInPortableRestoreMode` in `vmInfo._isNonPortableRestoreMode` when the `VM_getVMInfo` message is requested, and the result of this query doesn't change across a checkpoint/restore.

However, you do need to override the `isPortableRestoreModeEnabled` in `TR_J9ServerVM` in a manner similar to `isSnapshotModeEnabled()` where we don't need an explicit message, but there is a `TR_J9ServerVM` implementation:
```
bool
TR_J9ServerVM::isSnapshotModeEnabled()
   {
   JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
   auto *vmInfo = _compInfoPT->getClientData()->getOrCacheVMInfo(stream);
   return vmInfo->_isSnapshotModeEnabled;
   }
```
FWIW, it looks like the reason we have an explicit message for `VM_inSnapshotMode` is because the result of `inSnapshotMode` does change across a checkpoint/restore.